docs/reference/components/sources/postgresql_metrics.cue

```diff
@@ -0,0 +1,385 @@
+			"x86_64-apple-darwin":        true
+			"x86_64-pc-windows-msv":      true
+			"x86_64-unknown-linux-gnu":   true
+			"x86_64-unknown-linux-musl":  true
+		}
+
+		requirements: []
+
+		warnings: []
+		notices: []
+	}
+
+	installation: {
+		platform_name: null
+	}
+
+	configuration: {
+		endpoints: {
+			description: "PostgreSQL server endpoint in libpq-style connection strings."
+			required:    true
+			type: array: {
+				items: type: string: examples: ["postgresql://postgres:vector@localhost:5432/postgres"]
+			}
+		}
+		scrape_interval_secs: {
+			description: "The interval between scrapes."
+			common:      true
+			required:    false
+			type: uint: {
+				default: 15
+				unit:    "seconds"
+			}
+		}
+		namespace: {
+			description: "The namespace of metrics. Disabled if empty."
+			common:      false
+			required:    false
+			type: string: default: "postgresql"
+		}
+		included_databases: {
+			description: "A list of databases to match against `datname` column for which you want to collect logs from."
+			common:      false
+			required:    false
+			type: array: {
+				default: null
+				items: type: string: examples: ["postgres", "vector"]
+			}
+		}
+		excluded_databases: {
+			description: "A list of databases to match against `datname` column for which you don't want to collect logs from."
+			common:      false
+			required:    false
+			type: array: {
+				default: null
+				items: type: string: examples: ["postgres", "vector"]
+			}
+		}
+		tls: {
+			common:      false
+			description: "TLS options to connect to the PostgreSQL Server."
+			required:    false
+			type: object: {
+				examples: []
+				options: {
+					ca_file: {
+						description: "Path to CA certificate file."
+						required:    true
+						warnings: []
+						type: string: {
+							examples: ["certs/ca.pem"]
+						}
+					}
+				}
+			}
+		}
+	}
+
+	how_it_works: {
+		privileges: {
+			title: "Required Privileges"
+			body: """
+				PostgreSQL Metrics component collect metrics by making queries
+				to PostgreSQL Server. You need to be sure that provided user
+				allowed to make select queries for statistics views:
+
+				- `pg_stat_database`
+				- `pg_stat_database_conflicts`
+				- `pg_stat_bgwriter`
+				"""
+		}
+	}
+
+	telemetry: metrics: {
+		collect_completed_total:      components.sources.internal_metrics.output.metrics.collect_completed_total
+		collect_duration_nanoseconds: components.sources.internal_metrics.output.metrics.collect_duration_nanoseconds
+		request_errors_total:         components.sources.internal_metrics.output.metrics.request_errors_total
+	}
+
+	output: metrics: {
+		// Default PostgreSQL tags
+		_postgresql_metrics_tags: {
+			endpoint: {
+				description: "PostgreSQL endpoint."
+				required:    true
+				examples: ["postgresql:///postgres?host=localhost&port=5432"]
+			}
+			host: {
+				description: "The hostname of the PostgreSQL server."
+				required:    true
+				examples: [_values.local_host]
```

Comment:
Is `_values.local_host` correct here? Shouldn't this just be `"localhost"`?

Reply:
Yes, this is correct. `_values.local_host` refer to https://github.com/timberio/vector/blob/3b415d160de76e1ee4b600be5b475b922d38c221/docs/reference.cue#L14